packages/admin-e2e-tests/src/specs/homescreen/activity-panel.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { createSimpleProduct, withRestApi } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import { Login } from '../../pages/Login';
+import { OnboardingWizard } from '../../pages/OnboardingWizard';
+import { WcHomescreen } from '../../pages/WcHomescreen';
+import { createOrder, removeAllOrders, updateOption } from '../../fixtures';
+import { OrdersActivityPanel } from '../../elements/OrdersActivityPanel';
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { afterAll, beforeAll, describe, it } = require( '@jest/globals' );
+/* eslint-enable @typescript-eslint/no-var-requires */
+
+const simpleProductName = 'Simple order';
+const testAdminHomescreenActivityPanel = () => {
+	describe( 'Homescreen activity panel', () => {
+		const profileWizard = new OnboardingWizard( page );
+		const homeScreen = new WcHomescreen( page );
+		const ordersPanel = new OrdersActivityPanel( page );
+		const login = new Login( page );
+
+		beforeAll( async () => {
+			await login.login();
+
+			// This makes this test more isolated, by always navigating to the
+			// profile wizard and skipping, this behaves the same as if the
+			// profile wizard had not been run yet and the user is redirected
+			// to it when trying to go to wc-admin.
+			await withRestApi.deleteAllProducts();
+			await removeAllOrders();
+			await updateOption( 'woocommerce_task_list_hidden', 'no' );
```

Comment:
I might have to put this in a helper function, given we might use this more often.